### ISSUE TITLE
Design prompt to get the teacher

### ISSUE THREAD

**Issue by ehamner** [CMU-CREATE-Lab/preK-emotions-app#66]
If kid continues to feel sad, mad, etc. include a prompt to get the teacher. I found it in the wireframes but not in the later designs. (Might have been in Marvel when that was up?)

![wireframegetteacher](https://user-images.githubusercontent.com/24419423/53830948-923ec980-3f51-11e9-97bd-85a6047a1196.PNG)

I'm not sure what the screen should say if they click on the Get Teacher icon. Or what the interaction would be - are they still guided through the other steps or does it tell them to share their feelings with their teacher and then reset back to the beginning? @rachelleduffy @lczito @jenncross anyone remember talking about this in the focus groups?

**Comment by lczito** [comment]
It was supposed to become an option of a student went to do another activity, I believe

**Comment by ehamner** [comment]
Yes, I think so.

How do we think it would act once they click on it? It seems weird to tell them to let a teacher know how they are feeling, and then go on to the other steps like the heartbeat etc. I feel like it would be clearer if it directed them to a teacher and then went back to the screen where they select their face, so they could leave and find the teacher.

**Comment by lczito** [comment]
I agree!

**Comment by mo11yj** [comment]
Agreed, it should be an option with the "select another activity" page, not sure how that got lost, sorry!

And, agreed, I think it makes sense to redirect back to the choose your photo page. 

I can mock up the graphic for the tile to add to the page this weekend.

**Comment by mo11yj** [comment]
Thumbnail: zpl.io/VYkYBym
Updated screen with additional tile: zpl.io/VxLdeNR

<img width="1044" alt="Screen Shot 2019-03-09 at 11 13 03 AM" src="https://user-images.githubusercontent.com/25698542/54074090-5612ae00-425c-11e9-8dde-fbc7fc145e3b.png">

**Comment by rachelleduffy** [comment]
Can we actually have a picture of the classroom teacher? Remember...they
cannot read yet and they refer to the pictures for guidance....

**Comment by mo11yj** [comment]
I think we definitely can, we will have to add an option for the teacher to take their picture, and this illustration could be the default if they don't add an image. Does that seem like a good idea to everyone, and @tasota, is that reasonable to implement? Let me know and I can mock up how it can be added to the design.

**Comment by ehamner** [comment]
I think that might work in terms of the teachers being willing to add their own photo at least. The teachers using it currently supplied pictures so that they would appear with the class the same as the students. They used their own photos when they demonstrated.

**Comment by tasota** [comment]
With the teachers providing images this is definitely doable.

On a related note, would it be possible to also have some placeholder images, both for student thumbnails and for the teacher's thumbnail, as defaults? right now if we have no image for a student it just appears blank.

**Comment by mo11yj** [comment]
Added a place for teacher to upload their image here: 
zpl.io/bzN47gM, zpl.io/VQBZQj9

Uploading/choosing an image can use the same structure as other photo uploads in the app:
zpl.io/bAZ40Mn

Added thumbnail for student without photo here: 
zpl.io/VYkYBym

**Comment by tasota** [comment]
Thank you Molly!

How are we imagining the interaction for when the student selects the "talk to the teacher" option? Is there a prompt that appears or will it just go back to the list of students?

**Comment by tasota** [comment]
Nevermind, just read above. Sounds like we will just redirect to the list of students again. Thanks all!

**Comment by tasota** [comment]
We were discussing at our meeting today and realized it probably makes sense to have a static screen that prompts the student to get the teacher, and then it will go back to the list of students (without prompting about heartbeat and the rest).

@mo11yj could you design a screen for this? The text can read: "Talk to your teacher about how you are feeling"

**Comment by rachelleduffy** [comment]
Can it say it to the children? They cannot read yet...

**Comment by tasota** [comment]
@rachelleduffy Yep! All of the text throughout the app have audio prompts associated with them

**Comment by rachelleduffy** [comment]
Excellent!!! Thank you!!!

**Comment by mo11yj** [comment]
If a teacher has uploaded a photo: zpl.io/br8G8DW
If not: zpl.io/aMPXPrd

**Comment by ehamner** [comment]
@mo11yj , @lczito and I think the teacher image might be too big. Neither of us wanted our face to be that big on the screen so we thought the teachers wouldn't like it either. Can you make us a design where their picture is smaller? I thought maybe it could be like the generic background, but with some more space between the two people where the teacher's photo would go. What do you think?

**Comment by mo11yj** [comment]
How about this?

zpl.io/br8G8DW


<img width="1047" alt="Screen Shot 2019-03-31 at 2 11 22 PM" src="https://user-images.githubusercontent.com/25698542/55292976-e311d880-53be-11e9-8a62-511051ab8536.png">

**Comment by ehamner** [comment]
That works for me :)
